example/operator/20-garden.yaml

```diff
@@ -66,6 +66,28 @@ spec:
         enabled: false
   # volume:
   #   minimumSize: 20Gi
+  # Following options allow monitoring for certificate expiration dates
+  # Workergroup names are user defined
+  # workerGroups:
+  #   default:
```

Comment:
See https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#lists-of-named-subobjects-preferred-over-maps